library/std/unstable_arithmetic_internal.qs

```diff
@@ -404,4 +414,150 @@ namespace Microsoft.Quantum.Unstable.Arithmetic {
         }
     }
 
+    //
+    // Internal operations for comparisons
+    //
+
+    internal operation CompareGreaterThanOrEqualConstant<'T>(
```

Comment:
Even though these are internal operations, it would be nice to have a summary of what it does since it is fairly complex. Additionally, it would be helpful to name the input parameters to reflect what they represent.

Reply:
Maybe this could be named `ApplyActionIfGreaterThanOrEqualConstant`?

Reply:
I added summary and description of the parameters. Also renamed function as you suggested.